### PR TITLE
Nrapstin/rstudio

### PR DESCRIPTION
**Pull request by natalya-patrikeeva**
page about RStudio

### CODE REVIEW

File: docs/_policies/leavingGSB.md [hunk 1]
```diff
@@ -42,18 +52,18 @@ To provide the best experience for current GSB researchers, the Yen systems have
 
 This data can be restored on demand, but depending on the data size, may incur an additional fee.
 
-### Exceptions
+#### Exceptions
 
 Some projects use a disproportionately large amount of data - in these instances, our team may expedite the process of purging this data in the interest of maintaining space for active GSB users.  In these situations, we will work with data owners to ensure this does not result in unacceptable data loss.
```
Comment (alexstorer) [inline docs/_policies/leavingGSB.md:57]: In these situations, we will work with data owners to ensure this does not result in unacceptable data loss.
how about...
`In these situations, we will make our best effort to coordinate with data owners to prevent unexpected data loss.`

File: docs/_policies/leavingGSB.md [hunk 2]
```diff
@@ -62,14 +72,18 @@ For other systems across campus, the data retention and offboarding policies may
 
 ## AWS
 
-If you use [Athena](../services/athena.html), [Cloud Forest](../services/cloudForest.html) or a custom built cloud solution, it is your responsibility to transition data off of these systems before leaving the school.
+If you've set up a AWS through Universtiy Information Technology, please go to [their cloud storage page](https://stanford.service-now.com/it_services?id=sc_category&sys_id=e6cc5a8813b9aa0019813598d144b09b){target="_blank"} to make an inquiry.
 
-{% include warning.html content="Cloud Forest is not designed for storing data, and any important results should be immediately transferred off of these ephemeral machines." %}
+If you use Athena, Cloud Forest or a custom built cloud solution, it is your responsibility to transition data off of these systems before leaving the school.
+
+!!! warning
```
Comment (alexstorer) [inline docs/_policies/leavingGSB.md:79]: I think we can cut this warning for brevity. Nobody makes these anymore -- we can keep mention just in case, but we don't need a callout

File: docs/_policies/leavingGSB.md [hunk 3]
```diff
@@ -62,14 +72,18 @@ For other systems across campus, the data retention and offboarding policies may
...
 
 The following may be decommissioned and purged without notice after you leave the GSB:
+
 * Cloud Forest instances and data
 * Custom built applications, services or websites
-* Accounts provisioned specifically for you
+* Accounts provisioned specifically for you by DARC
 * Unclaimed data on S3 or EBS volumes
 
 ### Sponsorship
```
Comment (alexstorer) [inline docs/_policies/leavingGSB.md:89]: This doesn't seem relevant to this page? I think we should cut this whole section

File: docs/_user_guide/best_practices_migrating_from_jupyter.md [hunk 4]
```diff
@@ -31,15 +38,13 @@ Before you migrate you have a few option
 * Settle for a Machin learning algorithm over deep learning. These can often times be as effective and use less resources.
```
Comment (alexstorer) [inline docs/_user_guide/best_practices_migrating_from_jupyter.md:38]: Typo, `machine`

Comment (alexstorer) [inline docs/_user_guide/best_practices_migrating_from_jupyter.md:38]: Typo above, `kernal`

Comment (alexstorer) [reply docs/_user_guide/best_practices_migrating_from_jupyter.md:38]: Also s/chatGPT/ChatGPT

Comment (alexstorer) [reply docs/_user_guide/best_practices_migrating_from_jupyter.md:38]: Also s/GPUS/GPUs

Comment (alexstorer) [inline docs/_user_guide/best_practices_migrating_from_jupyter.md:38]: Above: s/option/options

File: docs/_user_guide/best_practices_migrating_from_jupyter.md [hunk 5]
```diff
@@ -31,15 +38,13 @@ Before you migrate you have a few option
...
 
-!!! warning Not all algorithms or data types will scale memory linearly!
-
-<a href='/faqs/howCheckResourceUsage.html'>This article</a> gives some help on how to check your resources.  
-
+!!! warning
+    Not all algorithms or data types will scale memory linearly!
 
-# Migrating from JupyterHub to interactive Yens
+[`This article`](/_user_guide/best_practices_monitor_usage/){target="_blank"} gives some help on how to check your resources.
```
Comment (alexstorer) [inline docs/_user_guide/best_practices_migrating_from_jupyter.md:44]: Formatting -- why `this article` as code?

File: docs/_user_guide/best_practices_migrating_from_jupyter.md [hunk 6]
```diff
@@ -11,9 +18,9 @@ updateDate: 2023-12-18
 
 # Migrating Processes From JupyterHub to Yen-Slurm
```
Comment (alexstorer) [inline docs/_user_guide/best_practices_migrating_from_jupyter.md:19]: s/Yen-Slurm/`yen-slurm`

That's more consistent?

File: docs/_user_guide/best_practices_migrating_from_jupyter.md [hunk 7]
```diff
@@ -50,26 +55,27 @@ The biggest hurdle in migrating your process from a JupyterHub notebook to `yen-
 * Write a submit script to run your process on `yen-slurm` using your working environment
```
Comment (alexstorer) [inline docs/_user_guide/best_practices_migrating_from_jupyter.md:55]: "working environment" isn't clear

File: docs/_user_guide/best_practices_migrating_from_jupyter.md [hunk 8]
```diff
@@ -50,26 +55,27 @@ The biggest hurdle in migrating your process from a JupyterHub notebook to `yen-
...
 
 
-## Python Virtual Environments
+### Python Virtual Environments
 
-For Python, you can use  `venv` to create an environment that can be shared across Yen1-5, `yen-slurm`, and JupyterHub. See [this page](/topicGuides/pythonEnv.html) for information on setting up a Python virtual environment using `venv`.
+For Python, you can use  `venv` to create an environment that can be shared across Yen1-5, `yen-slurm`, and JupyterHub. See [this page](/_user_guide/best_practices_python_env/){target="_blank"} for information on setting up a Python virtual environment using `venv`.
```
Comment (alexstorer) [inline docs/_user_guide/best_practices_migrating_from_jupyter.md:60]: We link to this but WOW there's a lot of repeated information...do we have a strategy here for this?

File: docs/_user_guide/rstudio.md [hunk 9]
```diff
@@ -0,0 +1,20 @@
+# RStudio 
+
+On the Yens, RStudio Server is available via the JupyterHub. You can access your files and scripts and develop R code in a convenient environment. 
+
+!!! warning
+    RStudio only has a single version of R. Right now, it is a version 4.3.
```
Comment (alexstorer) [inline docs/_user_guide/rstudio.md:6]: s/a version/version

File: docs/_user_guide/rstudio.md [hunk 10]
```diff
@@ -0,0 +1,20 @@
...
+
+Launch RStudio from Launcher JupyterHub interface:
+![RStudio launch button from Jupyter](/assets/images/jupyter_rstudio.png)
+
+
+RStudio Server will open up in a new tab:
```
Comment (alexstorer) [inline docs/_user_guide/rstudio.md:12]: we need a warning/tip/whatever that this is real slow

**Review by alexstorer — CHANGES_REQUESTED** [review]

**Comment by natalya-patrikeeva** [comment]
Eh that's not right. Amy edited a bunch of these. My PR is supposed to be just the RStudio page
